### ISSUE TITLE
Don't export `jacobian`

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaApproximation/DomainSets.jl#87]
Conflicts with `ForwardDiff.jacobian`

**Comment by daanhb** [comment]
Yes good point. I've been thinking it might be better to put all of the maps-associated functionality in a separate submodule, and eventually in a separate package.

**Comment by dlfivefifty** [comment]
I would support that since it's orthogonal to domains

**Comment by daanhb** [comment]
Indeed. We identified a number of overlapping packages some time ago, but the majority of them don't support rectangular or other non-invertible maps so I went ahead and implemented what I needed to support e.g. parametric domains. But that should not be exposed when just using domains

**Comment by daanhb** [comment]
jacobian moved to FunctionMaps and will no longer be exported in 0.8
